tests/data/dcmdose/expected_dicom_patient_xyz.json

```diff
@@ -0,0 +1 @@
249999999999, 74.95249999999999, 76.95249999999999, 78.95249999999999, 80.95249999999999, 82.95249999999999, 84.95249999999999, 86.95249999999999, 88.95249999999999, 90.95249999999999, 92.95249999999999, 94.95249999999999, 96.95249999999999, 98.95249999999999, 100.95249999999999, 102.95249999999999], [-161.4519, -159.4519, -157.4519, -155.4519, -153.4519, -151.4519, -149.4519, -147.4519, -145.4519, -143.4519, -141.4519, -139.4519, -137.4519, -135.4519, -133.4519, -131.4519, -129.4519, -127.4519, -125.4519, -123.4519, -121.4519, -119.4519, -117.4519, -115.4519, -113.4519, -111.4519, -109.4519, -107.4519, -105.4519, -103.4519, -101.4519, -99.4519, -97.4519, -95.4519, -93.4519, -91.4519, -89.4519, -87.4519, -85.4519, -83.4519, -81.4519, -79.4519, -77.4519, -75.4519, -73.4519, -71.4519, -69.4519, -67.4519, -65.4519, -63.451899999999995, -61.451899999999995, -59.451899999999995, -57.451899999999995, -55.451899999999995, -53.451899999999995, -51.451899999999995, -49.451899999999995, -47.451899999999995, -45.451899999999995, -43.451899999999995, -41.451899999999995, -39.451899999999995, -37.451899999999995, -35.451899999999995, -33.451899999999995, -31.451899999999995, -29.451899999999995, -27.451899999999995, -25.451899999999995, -23.451899999999995, -21.451899999999995, -19.451899999999995, -17.451899999999995, -15.451899999999995, -13.451899999999995, -11.451899999999995, -9.451899999999995, -7.451899999999995, -5.451899999999995, -3.451899999999995, -1.4518999999999949, 0.5481000000000051, 2.548100000000005, 4.548100000000005, 6.548100000000005, 8.548100000000005, 10.548100000000005, 12.548100000000005, 14.548100000000005, 16.548100000000005, 18.548100000000005, 20.548100000000005, 22.548100000000005, 24.548100000000005, 26.548100000000005, 28.548100000000005, 30.548100000000005, 32.548100000000005, 34.548100000000005, 36.548100000000005, 38.548100000000005, 40.548100000000005, 42.548100000000005, 44.548100000000005, 46.548100000000005, 48.548100000000005, 50.548100000000005, 52.548100000000005, 54.548100000000005, 56.548100000000005, 58.548100000000005], [-1259.5, -1257.5, -1255.5, -1253.5, -1251.5, -1249.5, -1247.5, -1245.5, -1243.5, -1241.5, -1239.5, -1237.5, -1235.5, -1233.5, -1231.5, -1229.5, -1227.5, -1225.5, -1223.5, -1221.5, -1219.5, -1217.5, -1215.5, -1213.5, -1211.5, -1209.5, -1207.5, -1205.5, -1203.5, -1201.5, -1199.5, -1197.5, -1195.5, -1193.5, -1191.5, -1189.5, -1187.5, -1185.5, -1183.5, -1181.5, -1179.5, -1177.5, -1175.5, -1173.5, -1171.5, -1169.5, -1167.5, -1165.5, -1163.5, -1161.5, -1159.5, -1157.5, -1155.5, -1153.5, -1151.5, -1149.5, -1147.5, -1145.5, -1143.5, -1141.5, -1139.5, -1137.5, -1135.5, -1133.5, -1131.5, -1129.5, -1127.5, -1125.5, -1123.5, -1121.5, -1119.5, -1117.5, -1115.5, -1113.5, -1111.5, -1109.5, -1107.5, -1105.5, -1103.5, -1101.5, -1099.5, -1097.5, -1095.5, -1093.5, -1091.5, -1089.5, -1087.5, -1085.5, -1083.5, -1081.5, -1079.5, -1077.5, -1075.5, -1073.5, -1071.5, -1069.5, -1067.5, -1065.5, -1063.5, -1061.5, -1059.5]]}
```

Comment:
When making json files it's a good idea to include the indent parameter, say `indent=2`. Makes the file much more human readable :). 

Copy pasting this into http://jsonprettyprint.com should do this now that the file is already made though.

Reply:
I can adjust easily enough :)

Reply:
Another thing that really helps is `np.round(array, decimals=4).tolist()` instead of going straight to list. That fixes up those floating point deviations.